deploy-service/common/src/main/java/com/pinterest/deployservice/scm/PhabricatorManager.java

```diff
@@ -181,8 +182,8 @@ public String getUrlPrefix() {
     }
 
     @Override
-    public String getType() {
-        return TYPE;
+    public String getTypeName() {
+        return typeName;
     }
```

Comment:
There's no way (to my knowledge) of getting the value of typeName into the class as an argument if a BaseManager object were to be instantiated, which is not possible since it is an abstract class